Robot Code/src/org/usfirst/frc/team3316/robot/commands/chassis/RetractOmni.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc.team3316.robot.commands.chassis;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class RetractOmni extends DBugCommand
+{
+	private double timeout;
+
+	public RetractOmni()
+	{
+		requires(Robot.chassis); // We don't want the driver to stall the motors
+	}
+
+	protected void init()
+	{
+		// Closing requires no coordination, so yolo
```

Comment:
no coordination at all, not event short before long? I think it makes more sense mechanically - ask Ofer.